redisinsight/ui/src/components/formated-date/FormatedDate.tsx

```diff
@@ -0,0 +1,35 @@
+import { EuiToolTip } from '@elastic/eui'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { DATETIME_FORMATTER_DEFAULT, TimezoneOption } from 'uiSrc/constants'
+import { userSettingsConfigSelector } from 'uiSrc/slices/user/user-settings'
+import { formatTimestamp } from 'uiSrc/utils'
+
+export interface Props {
+  date: Date | string | number
+  truncate?: number
```

Comment:
`truncate` - the name for boolean variable. I suggest call it smth like `maxLength`

Reply:
followed your advice and moved truncate to css